Coq-Playground/README.md

```diff
@@ -0,0 +1,61 @@
+# Installation instructions for Coq
+
+The Coq HoTT library requires a bit of a customized installation of Coq.
+You can download pre-built packages for various platforms by following the links here:
+
+  - For Windows: [32 bits]() and [64 bits]()
+  - For [MacOS]()
+  - For Linux distributions: a [snap]() package is available.
+
+These packages come with an Integrated Development Environment, CoqIDE for short,
+which needs to be configured to call a specific Coq version. To do so, simply go 
+to the `Preferences` menu, under `Edit`, then the `External` tab, and set
+`coqidetop` to:
+    
+  On MacOS: `/Applications/Coq_Platform_2021.02.0.app/Contents/Resources/bin/hoqidetop` 
+  On Windows: `???`
+  On Linux: `/usr/local/bin/hoqidetop` (TODO: check)
```

Comment:
With the snap it should go to /snap/bin/coq.hoqidetop (you have to change the snapcraft.in at the very bottom).
With the scripts, it would be in ~/.opam/_coq_platform_hott_..../bin/hoqidetop

Reply:
Thanks!